### PR TITLE
Minimal wrist feed icons

### PR DESCRIPTION
**Pull request by Natsumi-sama**
Not sure how performant spamming VRCXStorage.GetBool() is and this clutters vr.pug a bit

### CODE REVIEW

File: html/src/vr.pug [hunk 1]
```diff
@@ -15,42 +15,80 @@ html
...
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    | ◀️ #[span.name(v-text="feed.data")]
-                        div(v-else-if="feed.type === 'Location'" class="x-friend-item" :class="{ friend: feed.isFriend, favorite: feed.isFavorite }")
-                            .detail
-                                span.extra
-                                    span.time {{ feed.created_at | formatDate('HH:MI') }}
-                                    location(:location="feed.data")
+                    template(v-if="VRCXStorage.GetBool('VRCX_minimalFeed')")
```
Comment (pypy-vrc) [inline html/src/vr.pug:18]: 😨

Comment (Natsumi-sama) [reply html/src/vr.pug:18]: yea... I'm sure there is a better way, maybe RefreshVR() on setting change?

**Comment by pypy-vrc** [comment]
it seems that the minimal feed's icon doesn't mean login/logout we need a clear icon

image

**Comment by pypy-vrc** [comment]
tip: You can open the VR overlay form by this way 😆

image

**Comment by Natsumi-sama** [comment]
yes I will look for some better icons.

I was using the form but I had it run with my debug version, this is an easier way though 👍

**Pull request merged**